### PR TITLE
 Pause all non visible flickity

### PR DESCRIPTION
**Pull request by nitriques**
We do not need to play the flickities that are not visible.
This saves a lot of resources.

Starts to play when visible if auto play is active.

Add a new autoPlayWhenVisible property to make sure the flickity never
starts.

---

I would encourage everybody to drop the scroll listener and use this magic new API.

### CODE REVIEW

**Comment by fhamon** [comment]
So we drop IE 11 support by default? 😀

**Comment by nitriques** [comment]
@fhamon Should be easy to polyfill, if the effect is important. In this case, the polyfill is not needed: IE will have crappy animations anyways.

File: src/com/flickity.js [hunk 1]
```diff
@@ -88,87 +88,106 @@
...
+				initWithMultipleCells(slider);
 				
 				App.callback(o.inited);
 			} else if (slider.find(o.cellSelector).length === 1) {
 				slider.addClass(o.abortedClass);
 				slider.find(o.cellSelector).addClass(o.selectedClass);
 				App.callback(o.aborted);
 			} else {
-				App.log('No flickity found or multiple flickity in the scope');
+				App.log('No flickity found cell found in the scope');
```
Comment (dominic-blain) [inline src/com/flickity.js]: I think there is too many "found" in this sentence..

Comment (nitriques) [reply src/com/flickity.js]: @dominic-blain Good catch 👍 Fixed in a0830f1

**Pull request merged**